src/api/lib/tasks/dev/maintenance.rake

```diff
@@ -0,0 +1,72 @@
+namespace :dev do
+  namespace :maintenance do
+    # Run this task with: rails dev:maintenance:all
+    desc 'Set a maintenance environment'
+    task all: :development_environment do
+      Rake::Task['dev:maintenance:project'].invoke
+      Rake::Task['dev:maintenance:request_with_incident_actions'].invoke
+    end
+
+    desc 'Create a maintenance project'
+    task project: :development_environment do
+      admin = User.get_default_admin
+      leap = Project.find_by(name: 'openSUSE:Leap:15.0')
+
+      update_project = create(:update_project, target_project: leap, name: "#{leap.name}:Update", commit_user: admin)
+      create(
+        :maintenance_project,
+        name: 'MaintenanceProject',
+        title: 'official maintenance space',
+        target_project: update_project,
+        maintainer: admin,
+        commit_user: admin
+      )
+    end
+
+    desc 'Create a request with maintenance incident actions'
+    task request_with_incident_actions: :development_environment do
+      require 'factory_bot'
```

Comment:
I think you have to require this also for the maintenance `:project` task, its also using the factories

Reply:
True, done.